src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13530,3 +13530,212 @@ def test_aks_check_network(self, resource_group, resource_group_location):
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @live_only()
```

Comment:
Seems you've committed the corresponding recording file and CI passed, it's not necessary to add this mark.

If you mean the feature is protected by some feature flag or toggle and cannot be bypassed via custom feature header, could remove the case from default test matrix used by cli runner. `src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json`

Reply:
live_only is required due to role assignment is not mocked